Makefile

```diff
@@ -37,6 +37,7 @@ checks:
 	./scripts/checks "$(PKGS)"
 
 run-build: deps checks clean
+	cd ${GOPATH}/src/github.com/contiv/netplugin && version/generate_version  && \
```

Comment:
@mapuri - was going over the unaddressed comments; very interesting, I didn't know about it.
For now I will keep the scripting because I can see what got generated in a file and was comfortable with that.